sklearn/linear_model/_base.py

```diff
@@ -501,6 +506,26 @@ def fit(self, X, y, sample_weight=None):
         self : returns an instance of self.
         """
 
+        if self.normalize != "deprecate":
+            if not self.normalize:
+                warnings.warn("'normalize' was deprecated in version 0.24"
+                              " and will be removed in 0.26.", FutureWarning)
+            else:
+                warnings.warn("'normalize' was deprecated in version 0.24"
+                              " and will be removed in 0.26."
+                              " If you wish to keep normalizing your data"
```

Comment:
I think that we should not use the term `normalizing` but instead:

```
If you wish to keep an equivalent behaviour, use a Pipeline with a
StandardScaler a preprocessing stage: model = make....
```
No need for the import